cli/constants.py

```diff
@@ -7,6 +7,7 @@
 PUBLIC_KEY_REGEXP = r'^[0-9a-f]{66}$'
 PUBLIC_KEY_ADDRESS_REGEXP = r'^[0-9a-f]{70}$'
 HEADER_SIGNATURE_REGEXP = r'^[0-9a-f]{128}$'
+SWAP_IDENTIFIER_REGEXP = r'^[a-f0-9]{64}$'
```

Comment:
Change according to alphabetical order.

Reply:
This constants not in alphabetical order. It should be fixed in another ticket for technical dept.